<?xml version="1.0" encoding="utf-8"?>

<launch>

  <node name="octomap_compare" pkg="octomap_compare" type="octomap_compare_node" output="screen">

    <param name = "max_vis_dist" value = "3.18" />
    <param name = "distance_threshold" value = "1.0" />
    <param name = "eps" value = "0.3" />
    <param name = "min_pts" value = "10" />
    <param name = "base_file" value = "/home/lorenwel/data/octomaps/base.bt" />
    <param name = "comp_file" value = "/home/lorenwel/data/octomaps/comp.bt" />

  </node>

</launch>